lib/generator.js

```diff
@@ -1,37 +1,176 @@
 'use strict';
 
+const common = require('metarhia-common');
+
+const primitives = [
+  'boolean',
+  'null',
+  'undefined',
+  'number',
+  'string',
+  'symbol',
+];
+
+const globalObjects = [
+  'Object',
+  'Array',
+  'Date',
+  'Function',
+  'Map',
+  'WeakMap',
+  'Set',
+  'WeakSet',
+  'Int8Array',
+  'Uint8Array',
+  'Uint8ClampedArray',
+  'Float32Array',
+  'Error',
+  'EvalError',
+  'TypeError',
+  'RangeError',
+  'SyntaxError',
+  'ReferenceError',
+];
+
+const types = [
+  ...primitives,
+  ...globalObjects,
+  'Primitive',
+];
```

Comment:
Why doesn't this list match the part of the list you pass to the `ALL_TYPES` `Set` constructor in the beginning of the `lib/introspection.js`?
Should this maybe be declared in one place only?